CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+- Fix: always run in multiservice mode, ignoring -multiservice (#725)
```

Comment:
A quick look at this line (without examining the issue 725), will make our users believe that the broker now always runs in multiservice mode.
Not exactly what you have fixed ...

Better comment:
Fix: The option '-multiservice' was ignored and the broker ran always in multiservice mode. Now the option is taken taken into account (Issue #725)